test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -74,6 +74,103 @@ public void TensorFlowTransformMatrixMultiplicationTest()
             }
         }
 
+        private class ShapeData
+        {
+            // Data will be passed as flat vector.
+            // Intended data shape [2,2], model shape [2, None]
+            [VectorType(4)]
+            public float[] TwoDim;
```

Comment:
>TwoDim [](start = 27, length = 6)

Is it possible to have [?] model shape?
I mean one dimensional case, would we handle it correctly?

Reply:
Good catch. I should actually add a test for that too.

---
In reply to: [255220988](https://github.com/dotnet/machinelearning/pull/2475#discussion_r255220988) [](ancestors = 255220988)